common_benchmark_suite/openxla/benchmark/comparative_suite/pt/scripts/generate_model_artifacts.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2023 The OpenXLA Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import argparse
+import importlib
+import pathlib
+import re
+import multiprocessing
+import shutil
+import sys
+import torch
+import torch_mlir
+
+from import_utils import import_torch_module_with_fx, import_torch_module
+
+# Add openxla dir to the search path.
+sys.path.insert(0, str(pathlib.Path(__file__).parents[5]))
+from openxla.benchmark import def_types
+from openxla.benchmark.comparative_suite.pt import model_definitions
+from openxla.benchmark.models import model_interfaces, utils
+
+
+def _generate_artifacts(model: def_types.Model, save_dir: pathlib.Path):
+  model_dir = save_dir.joinpath(model.name)
+  model_dir.mkdir(exist_ok=True)
+  print(f"Created {model_dir}")
+
+  try:
+    # Remove all gradient info from models and tensors since these models are inference only.
+    with torch.no_grad():
+      model_module = importlib.import_module(model.model_impl.module_path)
+      model_obj: model_interfaces.InferenceModel = model_module.create_model(
+          **model.model_parameters)
+      if model_obj.import_on_gpu and not torch.cuda.is_available():
+        raise RuntimeError("Model can only be exported on CUDA.")
+
+      model_obj = model_obj.to(dtype=model_obj.dtype)
```

Comment:
I think the model has been converted to the target type, so I guess we don't need to convert again here.
https://github.com/openxla/openxla-benchmark/blob/main/common_benchmark_suite/openxla/benchmark/models/pt/bert/bert_model.py#L95

Reply:
Ah didn't see that, thanks!